### PR TITLE
#588 Redo search view calls

### PR DESCRIPTION
**Pull request by labboy0276**
The backend PR https://github.com/thinktandem/poets-api/pull/604 needs to merged along side of this as the search view has changed,

### CODE REVIEW

File: pages/search/index.vue [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
-        :key="`result.view_node-${i}`"
+        :key="`${i}`"
       >
         <b-col md="12">
           <div class="result-type">
             {{ result.type }}
           </div>
           <div class="result-title">
-            <b-link :to="result.view_node">{{ result.title }}</b-link>
+            <span v-html="result.title" />
```
Comment (serundeputy) [inline pages/search/index.vue]: @labboy0276 looks like these do not link through to the nodes any longer.

Comment (pirog) [reply pages/search/index.vue]: nice catch @serundeputy, @labboy0276 ^

Comment (labboy0276) [reply pages/search/index.vue]: They do.  There is no link to content on a search views, this renders as an <a tag>  hence the v-html attribute.

Comment (labboy0276) [reply pages/search/index.vue]: @serundeputy @pirog

Comment (serundeputy) [reply pages/search/index.vue]: @labboy0276 gotchya i tested before the drupal side was merged; there are links now :+1: 

the `person` node does not seem to be prioritized though; which is what I thought kicked this whole thing off; https://pr-359-y2ihrpi-tcynzice66pk2.us-2.platformsh.site/search?combine=blanco

person node is 4th

**Comment by labboy0276** [comment]
Seems so @serundeputy I found an module to help and will put it on the other ticket

**Pull request merged**